rel/overlay/etc/default.ini

```diff
@@ -930,3 +930,97 @@ url = {{nouveau_url}}
 ;max_objects = 10000
 ;max_idle = 600000
 ;enable = true
+
+[couch_scanner]
+; How often to check for configuration changes and start/stop plugins
+;interval_sec = 5
+
+; Minimum time to force a plugin to wait before running again after a crash
+;min_penalty_sec = 30
+
+; Maximum time to force a plugin to wait after repeated crashes (8 hours default)
+;max_penalty_sec = 28800
+
+; If plugin runs successfully without crashing for this long, reset its
+; repeated error count
+;heal_threshold_sec = 300
+
+; Database processing rate limit. This will also be the rate at which design
+; documents are fetched. The rate is shared across all running plugins.
+;db_rate_limit = 50
```

Comment:
What is the time unit this operates on? Per second?

Reply:
Yeah, on average. It may burst for 5 seconds as that's how often we refill the token "buckets". I'll update the comment around the values to indicate it's an average rate per second.